_data/whats-new.yml

```diff
@@ -4,8 +4,41 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Apr 29 09:56:29 2019
+updated: Mon May  6 15:32:53 2019
 entries:
+- description: Added a [new code example](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_dialog.html)
+    showing how to initialize and pass options to the dropdown dialog widget.
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: May  3, 2019
+  link: https://github.com/magento/devdocs/pull/4346
+- description: Added Argument type examples in [Layout instructions](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: May  3, 2019
+  link: https://github.com/magento/devdocs/pull/4333
+- description: Added a code example section to [Collapsible Widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_collapsible.html).
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: May  2, 2019
+  link: https://github.com/magento/devdocs/pull/4351
+- description: Added information about the Magento Coding Standard to the following
+    topics:<br/>- [Code sniffers](https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-sniffers.html)<br/>-
+    [PHP coding standard](https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-php.html)<br/>-
+    [Template CSS security](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-security.html)<br/>-
+    [Security, performances, and data handling](https://devdocs.magento.com/guides/v2.3/ext-best-practices/extension-coding/security-performance-data-bp.html)<br/>-
+    [Working with architecture](https://devdocs.magento.com/guides/v2.3/ext-best-practices/extension-coding/working-with-arch-bp.html)<br/>-
+    [Test your component](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/validate/test-module.html)
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: May  2, 2019
+  link: https://github.com/magento/devdocs/pull/4360
+- description: Page Builder migration documentation - Learn how to use the Page Builder
+    migration module to [migrate your BlueFoot content to Page Builder](https://devdocs.magento.com/page-builder-migration/docs/).
+  versions: 2.3.1
+  type: New topic
+  date: May  2, 2019
```

Comment:
Is there an extra space?

Reply:
yes, it is automatically generated by date converter

Reply:
fixed this in the generator dshevtsov/whats-new-on-devdocs@4022a93